### ISSUE TITLE
[Bug]: Error launching game with jei-1.21.1-fabric-19.17.0.192.jar

### ISSUE THREAD

**Issue by ST-DDT** [mezz/JustEnoughItems#3751]
### Steps to Reproduce the Bug

Install JEI
Start Minecraft

### Expected Behavior

Game starts without issues

### Actual Behavior

> Mod 'Just Enough Items' (jei) 19.17.0.192 requires version **fabricLoaderVersionRange** of mod 'Fabric Loader'(fabricloader), but only the wrong version is present: 0.16.5!

### Extra Notes (Optional)

The variables haven't been replaced

````json
  "depends": {
    "fabricloader": "fabricLoaderVersionRange",
    "fabric-api": "fabricApiVersionRange",
    "java": ">=21"
  }
````

expected

````json
  "depends": {
    "fabricloader": ">=0.16.3",
    "fabric-api": ">=0.103.0+1.21.1",
    "java": ">=21"
  }
````

works: jei-1.21.1-fabric-19.16.5.185.jar (Modrinth)
broken: jei-1.21.1-fabric-19.17.0.192.jar (Modrinth)

**Comment by Mace68** [comment]
Same error here with jei-1.21.1-fabric-19.17.0.192.jar from Modrinth also. jei-1.21.1-fabric-19.17.0.191.jar seems to work fine.

**Comment by mezz** [comment]
Thanks for the report!
I have just resolved this now, in version `19.17.0.193`.
